changelogs/unreleased/gh-5843-fiber-wakeup.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/core
+
+* Remove assertion raised in case of `fiber_wakeup()` get called with
+  dead fibers. Due to backward compatibility we've allowed such calls
+  for release builds but didn't not in debug builds. In result there
```

Comment:
```
[cyrill@grain tarantool.git] git diff
diff --git a/changelogs/unreleased/gh-5843-fiber-wakeup.md b/changelogs/unreleased/gh-5843-fiber-wakeup.md
index c0e7007b0..f501bc04f 100644
--- a/changelogs/unreleased/gh-5843-fiber-wakeup.md
+++ b/changelogs/unreleased/gh-5843-fiber-wakeup.md
@@ -2,5 +2,5 @@
 
 * Remove assertion raised in case of `fiber_wakeup()` get called with
   dead fibers. Due to backward compatibility we've allowed such calls
-  for release builds but didn't not in debug builds. In result there
+  for release builds but not for debug builds. In result there
   was inconsistency between program behaviour (gh-5843).
```

Reply:
Thanks a huge, Serge!